app/customer.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import annotations
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: dict,
```

Comment:
I think you could implement a class Location that will have coords as parameters on init and will have calculate_distance method. This will structure your code better